posts/how-to-search-code-with-sourcegraph-a-cheat-sheet.md

```diff
@@ -25,10 +25,17 @@ By default, Sourcegraph lets you search globally, providing you with results fro
 
 The `repo` keyword lets you search a specific repository in your organization or on the web.
 
-```
-repo:[repository-path]
-repo:[regular-pattern]
-```
+<PrismSyntaxHighlighter
+input={`repo:[regular-pattern]`}
+language='bash'
+matcher='[repository=path]'
+/>
+
+<PrismSyntaxHighlighter
+input={`repo:[regular-pattern]`}
+language='bash'
+matcher='[regular-pattern]'
+/> 
```

Comment:
Opted for doing two blocks to enable highlighting of more than one line, which I think we did not consider when building the highlighter tool. Maybe a list being passed to `matcher` would help this?

Reply:
Had to change `PrismSyntaxHighlighter` to `Highlighter` due to regular expression matching issues